packages/wp-now/src/wp-now.ts

```diff
@@ -163,7 +164,11 @@ async function runWpContentMode(
 	await initWordPress(php, wordPressVersion, documentRoot, absoluteUrl);
 	fs.ensureDirSync(wpContentPath);
 
-	php.mount(projectPath, `${documentRoot}/wp-content`);
+	if (hasPluginOrThemeDirectory(projectPath)) {
+		php.mount(projectPath, `${documentRoot}/wp-content`);
+	} else {
+		php.mount(projectPath, documentRoot);
+	}
```

Comment:
@sejas Any ideas why the Learn `wp-content` directory isn't being mounted correctly?

<img width="1506" alt="image" src="https://github.com/WordPress/playground-tools/assets/36432/b3be60aa-39a9-4065-8fcf-948e25222ce4">

Reply:
As it turns out, it was following the first part of the conditional because there were some hidden directories (https://github.com/WordPress/playground-tools/issues/32):

<img width="1888" alt="image" src="https://github.com/WordPress/playground-tools/assets/36432/07647a32-483e-4a3c-94c8-1f573b1e577b">

The hidden directories existed because I previously ran `wp-now start` in the directory.

When I remove the hidden directories, I see this crash:

```
Error: Could not mount /Users/danielbachhuber/projects/wordpress-learn: Device or resource busy.
    at _NodePHP.descriptor.value (/Users/danielbachhuber/projects/playground-tools/node_modules/@php-wasm/node/index.cjs:67405:17)
    at runWpContentMode (file:///Users/danielbachhuber/projects/playground-tools/dist/packages/wp-now/main.js:590:9)
    at async file:///Users/danielbachhuber/projects/playground-tools/dist/packages/wp-now/main.js:535:9
    at async applyToInstances (file:///Users/danielbachhuber/projects/playground-tools/dist/packages/wp-now/main.js:481:5)
    at async startWPNow (file:///Users/danielbachhuber/projects/playground-tools/dist/packages/wp-now/main.js:532:3)
    at async startServer (file:///Users/danielbachhuber/projects/playground-tools/dist/packages/wp-now/main.js:780:42)
    at async Object.handler (file:///Users/danielbachhuber/projects/playground-tools/dist/packages/wp-now/main.js:925:25) {
```

Reply:
With more reflection, I kinda think this particular scenario would be better solved by https://github.com/WordPress/playground-tools/issues/19

Reply:
> @sejas Any ideas why the Learn `wp-content` directory isn't being mounted correctly?
@danielbachhuber , I debugged the code and `php.mount(projectPath, documentRoot);` is trying to mount twice the same destination`documentRoot`. This directory was already mounted with the WordPress latest files.

A simple suggestion would be mounting the `wp-content` folders.
```
- php.mount(projectPath, documentRoot);
+ php.mount(`${projectPath}/wp-content`, `${documentRoot}/wp-content`);
```
But this approach would be the "same" as running wp-now directly from inside the `wp-content` folder.

A more elaborated approach would be mounting each subfolder/file independently resulting on a "merge" between core and the plugin, maybe even recursively if we want that `themes` has twenty.


This is what I see with the simple approach:
### Themes:
<img width="1691" alt="Screenshot 2023-06-02 at 19 22 32" src="https://github.com/WordPress/playground-tools/assets/779993/c5ad5b75-a402-47f5-bb04-8c14ecec3c02">


### Plugins:
<img width="1151" alt="Screenshot 2023-06-02 at 19 22 47" src="https://github.com/WordPress/playground-tools/assets/779993/79ec11fa-39e3-4e14-9e8e-08ee51da842e">

Reply:
I already pushed the simple fix: https://github.com/WordPress/playground-tools/pull/51/commits/1b2260fb6b5ded4f65fd66ece874f62ed7efb35a

Reply:
@sejas Thanks for taking a look at this!

I think I'm leaning towards _not_ shipping this pull request, and instead solving this with https://github.com/WordPress/playground-tools/issues/19

I'm not strongly opinionated about it, though. What do you think?

Reply:
Mapping folders is a more general fix, let's go for it. 👍